pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -540,46 +324,3 @@ def get_active_optimizers(self, batch_idx: Optional[int] = None) -> List[Tuple[i
         # find optimzier index by looking for the first {item > current_place} in the cumsum list
         opt_idx = int(np.argmax(self.optimizer_freq_cumsum > current_place_in_loop))
         return [(opt_idx, self.trainer.optimizers[opt_idx])]
-
-    def _build_kwargs(self, batch: Any, batch_idx: int, opt_idx: int, hiddens: Optional[Tensor]) -> Dict[str, Any]:
```

Comment:
reviewer: moved to utilities to share code with OptimizerLoop, split out in #9198